tensorboard/webapp/widgets/linked_time_fob/linked_time_fob_controller_component.ts

```diff
@@ -194,4 +194,31 @@ export class LinkedTimeFobControllerComponent {
     // No fobs can pass the lowest step in this graph.
     return 0;
   }
+
+  stepTyped(fob: Fob, step: number) {
+    let newLinkedTime = this.linkedTime;
```

Comment:
Not sure what the intention here is but "newLinkedTime" is now the same as "this.linkedTime" and any edits to the former will modify the latter.

Did you mean

```
let newLinkedTime = {...this.linkedTime};
```

Reply:
handled in summary comment.

Good catch. This did not matter since we emit an action that updates this.linkedTime anyway. But, this was definitely not intended.